Controller/ContentTypeController.php

```diff
@@ -232,7 +247,24 @@ public function createContentTypeAction(Request $request, $contentTypeGroupId, $
         $createForm = $this->createForm(new ContentTypeCreateType($this->contentTypeService), ['contentTypeGroupId' => $contentTypeGroupId]);
         $createForm->handleRequest($request);
         if ($createForm->isValid()) {
-            $languageCode = $languageCode ?: $this->prioritizedLanguages[0];
+            $verifiedLanguageCodes = $this->getVerifiedLanguageCodes(
```

Comment:
I think we can simplify, assuming:
- ui should provide drop down of languages to create in based on data in LanguageService, so that is already verified
- if prioritizedLanguages[] is wrong, it will fail later, however it a mistake of developer that he can fix, so we can live with that.

then the logic can be simplified to be what is was potentially, however lest take a skype call on this, I need to get the bigger picture :)